src/studio/nuxeo-studio/nuxeo-studio-release-notes/2017-10-10-studio-71.md

```diff
@@ -0,0 +1,25 @@
+---
+title: October 10, 2017
+description: This version focuses on some speed optimization. Branches listing loading time improvement, ability to remove old releases.
```

Comment:
`This version focuses on speed optimization: improvement of branch list loading time; ability to remove old releases.`